__tests__/app.test.js

```diff
@@ -110,8 +110,28 @@ describe('GET /api/articles/:article_id', () => {
     })
            
 
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: gives you comment info based on article_id in path', () => {
+        return request(app)
+        .get('/api/articles/5/comments')
+        .expect(200)
+        .then(({body: {comments}}) => {
+            expect(comments).toHaveLength(2)
+            comments.forEach((comment) => {
+                expect(comment).toEqual(
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        comment_id: expect.any(Number),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        body: expect.any(String)
+                    })
+                )
```

Comment:
Also need tests checking that:
- the comments are coming back in most recent first order (descending by date)
- 200 response when it's a valid request but the article has no comments associated with it
- 400 for invalid ID e.g. '/api/articles/banana/comments'
- 404 for non-existent ID e.g.  '/api/articles/999999/comments'